gateway_addon/database.py

```diff
@@ -0,0 +1,80 @@
+"""Wrapper around the gateway's database."""
+
+import json
+import os
+import sqlite3
+
+
+_DB_PATHS = [
+    os.path.join(os.path.expanduser('~'),
+                 'mozilla-iot',
+                 'gateway',
+                 'db.sqlite3'),
+    os.path.join(os.path.expanduser('~'),
+                 '.mozilla-iot',
+                 'config',
+                 'db.sqlite3'),
```

Comment:
Is there any reason this can't include a relative path given that addons are almost always run from `build/addons`?

Reply:
I typically run out of tree when I'm developing add-ons. Also, relative paths won't be reliable if the add-ons change their current directory. There's an optional `path` parameter that can be passed to `Database()` that should let devs do whatever they want, though.

Reply:
My main constraint is that I want any python add-on I download to work out of the box on my laptop where the gateway is in a slightly nonstandard location. The unified config location fixes this for the future but if adding '../../../db.sqlite3' works for 90% of custom setups I think we should have it.

Reply:
Fair enough. Done.